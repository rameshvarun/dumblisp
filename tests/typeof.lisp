(print
  (typeof 3)
  (typeof "hello")
  (typeof ())
  (typeof (list 3 3))
  (typeof 'sym)
  (typeof typeof)
  (typeof (lambda () 3)))
