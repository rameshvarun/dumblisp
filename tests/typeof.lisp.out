integer string list list symbol builtin function
